Source/WebCore/svg/SVGLengthContext.cpp

```diff
@@ -88,6 +88,20 @@ float SVGLengthContext::resolveLength(const SVGElement* context, SVGUnitTypes::S
     return x.valueAsPercentage();
 }
 
+static inline float dimensionForLengthMode(SVGLengthMode mode, std::optional<FloatSize> viewportSize)
```

Comment:
This should just be FloatSize viewportSize.

Reply:
I tried `const FloatSize&` but it gives compile error.

Reply:
FloatSize should be sufficient, but you need to update 2 of the callsites too (*viewportSize).

Reply:
On it. Testing locally.

Reply:
Done and pushed update with feedback. :-)